.github/workflows/build-docker-full.yml

```diff
@@ -16,15 +16,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out the repo
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         id: setup-node
         with:
           node-version: ${{ vars.NODE_VERSION_OVERRIDE || 'lts/*' }}
 
       - name: Log in to Docker Hub
-        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        uses: docker/login-action@40891eba8c2bcd1309b07ba8b11232f313e86779
```

Comment:
why pin to this commit?
it's from April 17:
https://github.com/docker/login-action/tree/40891eba8c2bcd1309b07ba8b11232f313e86779

but doesn't match any of the last releases:
https://github.com/docker/login-action/releases

Reply:
Because I was doing manual code reviews of these GHA commits but this was already being used in code builder https://github.com/forcedotcom/code-builder-images/blob/5f6d2307bee335d763ef408d915fdf4dcf500c81/.github/workflows/release-images.yml#L79. I used it because it accomplished the goal of removing the warnings and was already in use